cli_tools/gce_ovf_import/main.go

```diff
@@ -69,6 +69,7 @@ var (
 	hostname                    = flag.String(ovfimportparams.HostnameFlagKey, "", "Specify the hostname of the instance to be created. The specified hostname must be RFC1035 compliant.")
 	machineImageStorageLocation = flag.String(ovfimportparams.MachineImageStorageLocationFlagKey, "", "GCS bucket storage location of the machine image being imported (regional or multi-regional)")
 	buildID                     = flag.String("build-id", "", "Cloud Build ID override. This flag should be used if auto-generated or build ID provided by Cloud Build is not appropriate. For example, if running multiple imports in parallel in a single Cloud Build run, sharing build ID could cause premature temporary resource clean-up resulting in import failures.")
+	computeServiceAccount       = flag.String("compute_service_account", "", "Compute service account to be used by importer. When empty, the Compute Engine default service account is used.")
```

Comment:
Is this PR implementing compute service account for OVF import as well? I thought it's only image/one-step import?

Btw, OVF importer uses "-" instead of "_" in flags (should be `compute-service-account`).